cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1388,28 +1388,28 @@ def test_advanced_problems(self):
         Test the handling of advanced problem templates.
         """
         problem_templates = self.get_templates_of_type('problem')
-        ora_template = self.get_template(problem_templates, u'Peer Assessment')
-        self.assertIsNotNone(ora_template)
-        self.assertEqual(ora_template.get('category'), 'openassessment')
-        self.assertIsNone(ora_template.get('boilerplate_name', None))
+        circuit_template = self.get_template(problem_templates, u'Circuit Schematic Builder')
+        self.assertIsNotNone(circuit_template)
+        self.assertEqual(circuit_template.get('category'), 'problem')
+        self.assertEqual(circuit_template.get('boilerplate_name'), 'circuitschematic.yaml')
 
-    @patch('django.conf.settings.DEPRECATED_ADVANCED_COMPONENT_TYPES', ["combinedopenended", "peergrading"])
-    def test_ora1_no_advance_component_button(self):
+    @patch('django.conf.settings.DEPRECATED_ADVANCED_COMPONENT_TYPES', ["poll", "survey"])
+    def test_deprecated_no_advance_component_button(self):
         """
-        Test that there will be no `Advanced` button on unit page if `combinedopenended` and `peergrading` are
-        deprecated provided that there are only 'combinedopenended', 'peergrading' modules in `Advanced Module List`
+        Test that there will be no `Advanced` button on unit page if units are
+        deprecated provided that they are the only modules in `Advanced Module List`
         """
-        self.course.advanced_modules.extend(['combinedopenended', 'peergrading'])
+        self.course.advanced_modules.extend(['poll', 'survey'])
         templates = get_component_templates(self.course)
         button_names = [template['display_name'] for template in templates]
         self.assertNotIn('Advanced', button_names)
 
-    @patch('django.conf.settings.DEPRECATED_ADVANCED_COMPONENT_TYPES', ["combinedopenended", "peergrading"])
-    def test_cannot_create_ora1_problems(self):
+    @patch('django.conf.settings.DEPRECATED_ADVANCED_COMPONENT_TYPES', ["poll", "survey"])
+    def test_cannot_create_deprecated_problems(self):
         """
-        Test that we can't create ORA1 problems if `combinedopenended` and `peergrading` are deprecated
+        Test that we can't create problems if they are deprecated
         """
-        self.course.advanced_modules.extend(['annotatable', 'combinedopenended', 'peergrading'])
+        self.course.advanced_modules.extend(['annotatable', 'poll', 'survey'])
```

Comment:
out of scope for this PR, but maybe we should make a dummy deprecated xblock for the purposes of these tests. @symbolist , what do you think?